service/dynamodb/internal/expclient/main.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
This file seems to be an example file. Should this file be renamed to follow example file notation?

Reply:
This is an internal client that i used for debugging the customization.  I could deleted it completely, but it is not intended to be used by the user, only for our development/testing.